packages/client/src/comp/util/Signature.tsx

```diff
@@ -18,11 +20,13 @@ type Props = React.PropsWithChildren<{
 }>
 
 export const Signature: React.FC<Props> = ({ setSignature }) => {
+  const { search } = useLocation()
```

Comment:
We actually return `query` in `useAppHistory`, which is what I think you want to actually use.

Reply:
done